### PR TITLE
Enable url button for play apps only when play server actually started

### PR DESCRIPTION
**Pull request by Warry**

### CODE REVIEW

File: ui/app/assets/services/sbt/tasks.js [hunk 1]
```diff
@@ -157,6 +157,11 @@ define([
   }
 
   var playApplicationUrl = ko.observable();
+  var playServerStarted = ko.observable(false);
+  // Watch whenever any task ends, because we can't target specifically play stop
+  workingTasks.run.subscribe(function(v) {
+    if (!v) playServerStarted(false);
+  });
```
Comment (havocp) [inline ui/app/assets/services/sbt/tasks.js]: ko.computed can save a lot of spaghetti:
```
var playServerStarted = ko.computed(function() {
  return workingTasks.run() && playApplicationUrl() !== null;
});
```
You'd still have to do `playApplicationUrl(null)` when the run job stops though.

**Comment by henrikengstrom** [comment]
Can't get this to work properly. 
Tested with OSX/Chrome and Reactive Stocks.

To recreate:
1. Run app 
2. URL button becomes visible after a while but is not clickable
3. Stop app
4. Button becomes clickable after a while but since there is no running app the URL doesn't work

**Comment by henrikengstrom** [comment]
Retested and things seem fine now. Must have been something local on my machine that caused this?

File: ui/app/assets/services/sbt/tasks.js [hunk 2]
```diff
@@ -156,7 +156,14 @@ define([
     return false;
   }
 
-  var playApplicationUrl = ko.observable();
+  var playApplicationUrl = ko.observable(null);
+  var playServerStarted = ko.computed(function() {
+    return workingTasks.run() && playApplicationUrl() !== null;
+  });
+  // Watch whenever any task ends, because we can't target specifically play stop
+  workingTasks.run.subscribe(function(v) {
```
Comment (havocp) [inline ui/app/assets/services/sbt/tasks.js]: you can target play stop; it should be a BackgroundJobFinished event on the job ID returned from backgroundRun.

**Pull request merged**